src/internal.c

```diff
@@ -7779,8 +7779,8 @@ static const char cannedMacAlgoNames[] =
 #endif
 
 static const char cannedKexAlgoNames[] =
-#if !defined(WOLFSSH_NO_ECDH_SHA2_NISTP256_KYBER_LEVEL1_SHA256)
-    "ecdh-sha2-nistp256-kyber-512-sha256,"
+#if !defined(WOLFSSH_NO_ECDH_NISTP256_KYBER_LEVEL1_SHA256)
+    "ecdh-nistp256-kyber-512-sha256,"
```

Comment:
`ecdh-nistp256-kyber-512-sha256` should become `ecdh-nistp256-kyber-512r3-sha256-d00@openquantumsafe.org` as per https://github.com/open-quantum-safe/openssh/issues/136